modules/ethereum/src/finality.rs

```diff
@@ -214,106 +214,110 @@ pub(crate) fn ancestry<'a, S: Storage>(
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use crate::tests::{genesis, validator, validators_addresses, InMemoryStorage};
-	use crate::HeaderToImport;
+	use crate::mock::{custom_test_ext, genesis, validator, validators_addresses, TestRuntime};
+	use crate::{BridgeStorage, HeaderToImport};
 
 	#[test]
 	fn verifies_header_author() {
-		assert_eq!(
-			finalize_blocks(
-				&InMemoryStorage::new(genesis(), validators_addresses(5)),
-				&Default::default(),
-				(&Default::default(), &[]),
-				&Default::default(),
-				None,
-				&Header::default(),
-				0,
-			),
-			Err(Error::NotValidator),
-		);
+		custom_test_ext(genesis(), validators_addresses(5)).execute_with(|| {
+			assert_eq!(
+				finalize_blocks(
+					&BridgeStorage::<TestRuntime>::new(),
+					&Default::default(),
+					(&Default::default(), &[]),
```

Comment:
I'm confused as why this test returns `Error::NotValidator`, but I suspect is has to do with this. I think we should have a variable like this

```rust
// A valid author would be x, y, or z
let invalid_author: ... =  (&Default::default(), &[]);
```

to make it clear.

Reply:
Unrelated to this PR. Can be addressed in #111